src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/nowhitespacebeforecasedefaultcolon/InputNoWhitespaceBeforeCaseDefaultColon.java

```diff
@@ -0,0 +1,106 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace.nowhitespacebeforecasedefaultcolon;
+
+import com.puppycrawl.tools.checkstyle.api.TokenTypes;
+
+/*
+* Config = default
+*/
+public class InputNoWhitespaceBeforeCaseDefaultColon {
```

Comment:
It would be a good idea to some switch expressions in here, you can find some more challenging examples in https://github.com/checkstyle/checkstyle/blob/master/src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/grammar/java14/InputJava14SwitchExpression.java

Reply:
Done, please see recent changes and suggest any changes if required.

Reply:
I do not see any switch statements/ expressions with compound/ block statements, it would be good to add a few of them.

Reply:
Do mean nested switch kind of statements, I didn't get what you mean by compound/block statement?

Reply:
Like this:

```java
        switch(x) {
            case 1: {  // <- curly brace here, now this is a "block statement"
                // do stuff
                break;
            }
            default: {  // <- curly brace here, now this is a "block statement"
                // do other stuff
            }
        }
```

Reply:
Done. Sir, can you just please help me on you to write PR description for Travis to pass?

Reply:
Travis will not pass until  https://github.com/checkstyle/contribution/pull/471 is merged.

Reply:
All right thanks, Please check the updated examples. I will fix the CI.

Reply:
Resolved.